### DISCUSSION TITLE
[IntelliSense] Run without the need of tailwind.config.js

### DISCUSSION THREAD

**Discussion by tclzcja** [tailwindlabs/tailwindcss#15292] [Ideas]
Hi there, just tried the v4 beta and it seems like Tailwind v4 can run even without tailwind.config.js. So just wondering is there a way to make the IntelliSense run without having a tailwind.config.{js,cjs,mjs,ts,cts,mts} file in the folder? Thank you!

For further elaboration, this is stated in the current extension "Troubleshoot" section:

<img width="645" alt="Screenshot 2024-12-04 at 10 46 40 AM" src="https://github.com/user-attachments/assets/6c4dfc7d-392a-44d8-b466-71a2c386568b">

Maybe only number 2 is necessary to tell whether Tailwind has been installed?

**Comment by ghost** [comment]
I think you're supposed to be able to do this by pointing the `tailwindCSS.experimental.configFile` option of the extension to your CSS file. https://github.com/tailwindlabs/tailwindcss-intellisense/?tab=readme-ov-file#tailwindcssexperimentalconfigfile

That got it partway there for me, though I don't have it all working yet.
